### PR TITLE
Add Nest Spec 

### PR DESCRIPTION
**Pull request by bmeverett**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behavior? (You can also link to an open issue here)**
No autocomplete support for nest

**What is the new behavior (if this is a feature change)?**
Adds autocomplete for the [Nest Cli](https://docs.nestjs.com/cli/overview)

**Additional info:**

### CODE REVIEW

File: src/nest.ts [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+      description: "Project in which to generate files",
+    },
+  },
+  {
+    name: "--flat",
+    description: "Enforce flat structure of generated element",
+  },
+  {
+    name: "--spec",
+    description: "Enforce spec files generation (default: true)",
```
Comment (fedeci) [inline src/nest.ts:21]:
```suggestion
    description: "Enforce spec files generation (default: true)",
    exclusiveOn: ["--no-spec"],
```

File: src/nest.ts [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+  },
+  {
+    name: "--no-spec",
+    description: "Disable spec files generation",
```
Comment (fedeci) [inline src/nest.ts:25]:
```suggestion
    description: "Disable spec files generation",
    exclusiveOn: ["--spec"],
```

File: src/nest.ts [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+    args: {
+      name: "collection",
+      description: "The collection name",
+    },
+  },
+];
+
+const completionSpec: Fig.Spec = {
+  name: "nest",
+  description: "",
```
Comment (fedeci) [inline src/nest.ts:39]: Could you add a description?

**Review by fedeci — CHANGES_REQUESTED** [review]
A couple tweaks and it should be ready to be merged in!

File: src/nest.ts (hunk 3 above)
Comment (bmeverett) [reply src/nest.ts]: Updated

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**